bigframes/functions/remote_function.py

```diff
@@ -654,9 +653,6 @@ def remote_function(
     output_type: Optional[type] = None,
     session: Optional[Session] = None,
     bigquery_client: Optional[bigquery.Client] = None,
-    bigquery_connection_client: Optional[
```

Comment:
This is a breaking change. Please detect if this is set and warn with a [FutureWarning](https://docs.python.org/3/library/exceptions.html#FutureWarning) that this argument is deprecated, instead.

Reply:
Thanks! Are we allowed to check in the breaking change? I do find it very confusing as a customer.

Reply:
I think the savior is that this is marked as deprecated for external use, so we can make a breaking change here?